components/feature/pwa/src/main/java/mozilla/components/feature/pwa/WebAppShortcutManager.kt

```diff
@@ -115,12 +116,23 @@ class WebAppShortcutManager(
             `package` = context.packageName
         }
 
+        val manifest = session.webAppManifest
+        val shortLabel = overrideShortcutName
```

Comment:
This is excellent use of the elvis operator! Although with every code branch, this increasing the missed code branches in coverage. Let's add tests to ensure we ensure this path is followed.